python-main.js

```diff
@@ -1629,6 +1629,10 @@ function web_editor(config) {
     // Join up the buttons in the user interface with some functions for
     // handling what to do when they're clicked.
     function setupButtons() {
+        if(navigator.platform.match('Win') !== null){
+            $(".roundsymbol").css("line-height","3.7rem");
+            $("#small-icons .status-icon").css("line-height","2.2rem");
```

Comment:
Instead of having the styles applied directly in the JS, could we have CSS classes in the CSS file named after the issue (roundtsymbol-firefox-win, or something like that) and then use JS to apply them?